var/spack/repos/builtin/packages/py-pytest/package.py

```diff
@@ -62,3 +64,6 @@ class PyPytest(PythonPackage):
     depends_on('py-importlib-metadata@0.12:', when='@4.6:5.0', type=('build', 'run'))
     depends_on('py-importlib-metadata@0.12:', when='@5.1: ^python@:3.7', type=('build', 'run'))
     depends_on('py-wcwidth', when='@4.5:', type=('build', 'run'))
```

Comment:
I think you need to get rid of this line, right? Otherwise you'll still depend on wcwidth in the latest version.

Reply:
Oops, of course. done